### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by danoscarmike**
- [x] Cloud Redis Link
- [x] Add BigQuery Data Transfer

### CODE REVIEW

**Comment by danoscarmike** [comment]
@quartzmo thanks for taking a look and likewise for the catch - I hadn't clocked that.  PTAL when you have a minute.

File: README.md [hunk 1]
```diff
@@ -500,12 +536,12 @@ subscriber.start
...
 ```
-### Redis API (Alpha)
+### Cloud Redis API (Alpha)
 
 - [google-cloud-redis README](google-cloud-redis/README.md)
-+- [google-cloud-redis API documentation](http://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud-redis/latest)
+- [google-cloud-redis API documentation](http://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud-redis/latest)
 - [google-cloud-redis on RubyGems](https://rubygems.org/gems/google-cloud-redis)
-- [Redis API documentation](https://cloud.google.com/redis/docs/)
+- [Redis API documentation](https://cloud.google.com/memorystore/docs/redis/)
```
Comment (quartzmo) [inline README.md]: @danoscarmike Shouldn't "[Redis API documentation]" be "[Cloud Redis API documentation]"?

Comment (blowmage) [reply README.md]: Great catch. Agreed.

Comment (blowmage) [reply README.md]: @danoscarmike Do you want to make this change or do you want one of us to push a commit to your PR?

**Comment by danoscarmike** [comment]
Thanks @quartzmo @blowmage - please feel free to make further changes and merge when ready

**Pull request merged**